CHANGES.md

```diff
@@ -9,6 +9,10 @@
 
 - Improve error message of cram test exceptions due to empty lines in a block (#270, @pitag-ha)
 
+#### Fixed
+
+- Fix the unit tests on Windows (#301 @tmattio)
```

Comment:
there's already a fixed section below. I think this should stick to just this work, ie "Use with-accepted-exit-codes in tests instead of system" - Windows is going to span more than a single PR

Reply:
Done - thanks!